build/repo.targets

```diff
@@ -69,7 +76,8 @@
 
     <ItemGroup>
       <PackageStoreManifestFiles Include="$(PackageCacheOutputPath)**\artifact.xml">
-        <DestinationFile>manifest.$(RID).xml</DestinationFile>
+        <TimestampDestinationFile>aspnetcore-store-$(TimestampVersion)-$(RID).xml</TimestampDestinationFile>
+        <NoTimestampDestinationFile>aspnetcore-store-$(NoTimestampVersion)-$(RID).xml</NoTimestampDestinationFile>
```

Comment:
Actually this isn't needed, we only produce the Timestamp package here. I need to update Coherence-Signed which hacks out a NoTimestamp version. https://github.com/aspnet/Coherence-Signed/blob/dev/dnx.msbuild#L482